.gitignore

```diff
@@ -17,3 +17,6 @@ spec/junit-reports
 .gem
 *.bulk
 vendor/
+
+.idea/
+log/
```

Comment:
Please explain why these are relevant to this PR.

Reply:
It's not relevant for the PR per se. I added them because they were useful for my development workflow. `./idea` is automatically created when using RubyMine. When running the tests the mounted volume from docker-compose created `log`. In my opinion, it makes sense to add it to gitignore. Feel free to commit these proposals directly to master. You seem to ask for a clean PR.

Reply:
It's fine 👍 Thank you for the explanation.